packages/devextreme/js/__internal/ui/chat/messagelist.ts

```diff
@@ -587,7 +592,8 @@ class MessageList extends Widget<Properties> {
       const $targetMessage = this._findMessageElementByKey(key);
 
       const bubble = MessageBubble.getInstance($targetMessage);
-      bubble.option('text', data.text);
+      const { text, isDeleted } = data;
```

Comment:
It works incorrectly. We shouldn't override text if it was existed and will not passed on update operation, for isDeleted as well